doc/intro.md

```diff
@@ -381,7 +371,7 @@ sum cancels out. Which leads to the following, much more compact block:
     I2(x2) | O4
            | O5
 
-Note that all transaction structure has been eliminated and the order of inputs and
+Note that all the transaction structure has been eliminated and the order of inputs and
```

Comment:
Here it is "transaction structure" rather than "_the_ transaction structure".
Analogous to "brain activity" and not "the brain activity".